build/prepare_deps.sh

```diff
@@ -6,6 +6,8 @@ OPUS_SHA=$2
 WHISPER_VERSION=$3
 WHISPER_SHA=$4
 MODELS=$5
+ONNX_VERSION=$6
+ONNX_SHA=$7
```

Comment:
You need to actually use this too 😜

Reply:
Just testing our security engineer's eye :p
Aha, good catch :)